link_shorterner/src/shortener_shortener.erl

```diff
@@ -0,0 +1,63 @@
+-module(shortener_shortener).
+
+-export([init/0,
+         short/1,
+         get/1,
+         subscribe/1,
```

Comment:
For a module this size it's ok, but if it grew a bit bigger, I would consider having a separate module that encapsulates the pg2 subscriptions code.